apis/Google.Cloud.OsLogin.V1Beta/Google.Cloud.OsLogin.V1Beta/ResourceNames.cs

```diff
@@ -0,0 +1,371 @@
+// Copyright 2017 Google LLC
```

Comment:
It looks like this has changed in Toolkit - I think you'll need to make a change to the copyright checker as well to make this pass CI.